userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,124 @@
+# One-Shot Object Detection
+
+One-Shot object detection is the task of learning to detect from as few as one example per category. Unlike the Object Detector which requires many varied examples of objects in the real world, the One-Shot detector only requires a single canonical example of an object to train the classifier on and performs predictions for that category in the wild. It is best suited for 2D objects or when there isn't much expected variance in the examples, such as playing cards, logos, road signs and clapperboards. One-Shot detector is not suitable for 3D objects such as faces, animals and cars. 
```

Comment:
I think we need to do a bit more explaining here to tell user what the difference is between the two.

Reply:
Opening another PR. The page didn't refresh and I missed noting the feedback above as well.